scripts/bigquery_schema_diff.py

```diff
@@ -287,6 +298,8 @@ def tmp_git(tmp_path: Path) -> Path:
 
     run(f"git clone {origin} {workdir}")
     os.chdir(workdir)
+    # make branches available by checking them out, but ensure state ends up on HEAD
```

Comment:
What happens if master isn't checked out?

Reply:
`git rev-parse` will run into `fatal: ambiguous argument 'master': unknown revision or path not in the working tree.` Running `git fetch --all` doesn't solve this issue for some reason.